src/5e-SRD-Magic-Items.json

```diff
@@ -783,15 +783,3069 @@
+    ],
+    "url": "/api/magic-items/dwarven-plate"
+  },
+  {
+    "index": "dwarven-thrower",
+    "name": "Dwarven Thrower",
+    "equipment_category": {
+      "name": "Weapon",
+      "url": "/api/equipment-categories/weapon"
+    },
+    "desc": [
+      "Weapon (warhammer), very rare (requires attunement by a dwarf)",
+      "You gain a +3 bonus to attack and damage rolls made with this magic weapon. It has the thrown property with a normal range of 20 feet and a long range of 60 feet. When you hit with a ranged attack using this weapon, it deals an extra 1d8 damage or, if the target is a giant, 2d8 damage. Immediately after the attack, the weapon flies back to your hand."
+    ],
+    "url": "/api/magic-items/dwarven-thrower"
+  },
+  {
+    "index": "efficient-quiver",
+    "name": "Efficient Quiver",
+    "equipment_category": {
+      "name": "Wondrous Item",
+      "url": "/api/equipment-categories/wondrous-item"
+    },
+    "desc": [
+      "Wondrous item, uncommon",
+      "Each of the quiver's three compartments connects to an extradimensional space that allows the quiver to hold numerous items while never weighing more than 2 pounds. The shortest compartment can hold up to sixty arrows, bolts, or similar objects. The midsize compartment holds up to eighteen javelins or similar objects. The longest compartment holds up to six long objects, such as bows, quarterstaffs, or spears.",
+      "You can draw any item the quiver contains as if doing so from a regular quiver or scabbard."
+    ],
+    "url": "/api/magic-items/efficient-quiver"
+  },
+  {
+    "index": "efreeti-bottle",
+    "name": "Efreeti Bottle",
+    "equipment_category": {
+      "name": "Wondrous Item",
+      "url": "/api/equipment-categories/wondrous-item"
+    },
+    "desc": [
+      "Wondrous item, very rare",
+      "This painted brass bottle weighs 1 pound. When you use an action to remove the stopper, a cloud of thick smoke flows out of the bottle. At the end of your turn, the smoke disappears with a flash of harmless fire, and an efreeti appears in an unoccupied space within 30 feet of you.",
+      "The first time the bottle is opened, the GM rolls to determine what happens.",
+      "| d100   | Effect                                                                                                                                                                                                                                                                                                                                                 |",
+      "|--------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|",
```

Comment:
Honestly though, you probably only need one space on each side and only three `-` for each column. This means you don't have to do this. Especially since this is just following Markdown style for the tables, which is nice and easy to use.

Reply:
Don't actually worry about this nitpick. I'll adjust it after it gets merged in.